card/src/main/res/layout/card_brand_item.xml

```diff
@@ -10,29 +10,31 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="wrap_content"
+    android:paddingTop="@dimen/standard_half_margin"
+    android:paddingBottom="@dimen/standard_half_margin">
 
     <com.adyen.checkout.ui.core.internal.ui.view.RoundCornerImageView
         android:id="@+id/imageView_cardBrandLogo"
-        style="@style/AdyenCheckout.Card.BrandSelection.Logo"
+        style="@style/AdyenCheckout.Card.BrandSelection.Item.Logo"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
     <androidx.appcompat.widget.AppCompatTextView
         android:id="@+id/textView_cardBrandName"
-        style="@style/AdyenCheckout.Card.BrandSelection.Title"
+        style="@style/AdyenCheckout.Card.BrandSelection.Item.Title"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintHorizontal_bias="0"
```

Comment:
I just noticed that this text should be `16sp`, but currently is `12sp`. Also this `layout_constraintHorizontal_bias` can be removed I think